packages/coverage-v8/src/browser.ts

```diff
@@ -0,0 +1,65 @@
+import { cdp } from '@vitest/browser/context'
+import type { V8CoverageProvider } from './provider'
+import { loadProvider } from './load-provider'
+
+interface ScriptCoverage {
+  result: Array<{ url: string }>
+}
+
+const session = cdp() as { send: (...options: any[]) => any }
```

Comment:
By the way, maybe this type casting is not needed if you add `@vitest/browser/playwright` to coverage-v8 tsconfig?